Library/Formula/cdrtools.rb

```diff
@@ -4,7 +4,10 @@ class Cdrtools < Formula
 
   stable do
     url "https://downloads.sourceforge.net/project/cdrtools/cdrtools-3.01.tar.bz2"
+    mirror "https://www.mirrorservice.org/sites/downloads.sourceforge.net/c/cd/cdrtools/cdrtools-3.01.tar.bz2"
+    mirror "https://fossies.org/linux/misc/cdrtools-3.01.tar.bz2"
     sha256 "ed282eb6276c4154ce6a0b5dee0bdb81940d0cbbfc7d03f769c4735ef5f5860f"
+    patch :p0, :DATA
```

Comment:
The patch for `isoinfo.c` fixes the bug in 3.01, but the one for `isoinfo.mk` is just a workaround for a bug in **`clang`**. This means we would still need to embed a patch in the formula. Now, since the loop that was triggering the bug in `clang` has been rewritten in 3.02a02 in such a way that `clang` does not produce wrong code anymore, it would be easy to get rid of the patch by simply adding a resource (to the 3.02a02 tarball) and steps (in a `if build.stable?` bloc) to overwrite `isoinfo.c` by a copy of the one in 3.02a02. I already know how to do this. May I change the formula for that? Or would you prefer to have an external patch for isoinfo.c and an embedded one in the formula?